app/revenue/utils.py

```diff
@@ -17,3 +17,9 @@
 along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 """
+import random
+import string
+
+
+def sku_generator(size=6, chars=string.ascii_uppercase + string.digits):
```

Comment:
If the SKU represents individually purchased items, especially the ad impressions, we should worry about collisions here with only 6 digits (will likely occur in under 100,000 instances)

Reply:
Reading the code over more I think the SKU refers to a category of item, with the `PlanItem` being the individually purchased item. If so, is there a reason to randomly generate these or should we just write human-readable SKUs?